cloudbuild/nightly.sh

```diff
@@ -105,6 +105,13 @@ case $STEP in
     exit
     ;;
 
+  deploy)
+    $MVN deploy -DskipTests -Dscala.skipTests=true -Prelease-nightly,dsv1_2.12,dsv1_2.13,dsv2_3.1,dsv2_3.2,dsv2_3.3,dsv2_3.4,dsv2_3.5,dsv2_4.0
```

Comment:
Please replace to the actual goal `deploy:deploy` instead of `deploy` (which is a maven lifecycle phase, and may run additional goals)